### ISSUE TITLE
As a developer, I'm able to specify the index name for DynamoDB tables.

### ISSUE THREAD

**Issue by bjinwright** [capless/kev#28]
See the example below. If the index_name is not specified then the backend would just use {field_name}-index like AWS does.
```python
class TestDocument(Document):
    name = CharProperty(required=True,unique=True,min_length=5,max_length=20,index_name='name_index')
    last_updated = DateTimeProperty(auto_now=True)
    date_created = DateProperty(auto_now_add=True)
    is_active = BooleanProperty(default_value=True,index=True,index_name='is_active_index')
    city = CharProperty(required=False,max_length=50)
    state = CharProperty(required=True,index=True,max_length=50)
    no_subscriptions = IntegerProperty(default_value=1,index=True,index_name='no_subs_index',min_value=1,max_value=20)
    gpa = FloatProperty()

    def __unicode__(self):
        return self.name
        

    class Meta:
        use_db = 'dynamodb'
        handler = kev_handler
```

**Comment by bjinwright** [comment]
@armicron

**Comment by bjinwright** [comment]
I think this can be achieved by changing get_index_name in the BaseDocument (kev.document.BaseDocument) class.

**Comment by armicron** [comment]
@bjinwright I think there need to change only this line https://github.com/capless/kev/blob/develop/kev/backends/dynamodb/db.py#L71

**Comment by bjinwright** [comment]
Yes, but I think you will need to make changes here https://github.com/capless/kev/blob/develop/kev/query.py and https://github.com/capless/kev/blob/develop/kev/document.py#L91.

**Comment by bjinwright** [comment]
Fixed in #31
